### PR TITLE
examples: Add custom labels example

### PR DESCRIPTION
**Pull request by ying-jeanne**
Fixes: https://github.com/prometheus/client_golang/issues/1585, tested in local. @bwplotka @ArthurSens @kakkoyun

### CODE REVIEW

File: examples/customlabels/main.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+)
+
+var addr = flag.String("listen-address", ":8080", "The address to listen on for HTTP requests.")
+
+func main() {
+	flag.Parse()
+
+	// Create a new registry.
+	reg := prometheus.NewRegistry()
+	prometheus.WrapRegistererWith(prometheus.Labels{"serviceName": "bartek"}, reg).MustRegister(
```
Comment (bwplotka) [inline examples/customlabels/main.go:36]:
```suggestion
	prometheus.WrapRegistererWith(prometheus.Labels{"serviceName": "my-service-name"}, reg).MustRegister(
```

**Review by bwplotka — APPROVED** [review]
Thanks! One nit, otherwise LGTM 👍🏽

**Comment by ying-jeanne** [comment]
> Thanks! One nit, otherwise LGTM 👍🏽

thanks for the review :), corrected so bad you don't like my service name.

**Review by ArthurSens — APPROVED** [review]
Thank you!

**Pull request merged**